frontend/packages/dev-console/integration-tests/features/user-preferences/user-preferences-dev-perspective.feature

```diff
@@ -0,0 +1,95 @@
+              And user selects "Developer" option
+             Then user can see "Developer" option selected in the perspective switcher
+
+
+        @regression @to-do
+        Scenario: Setting a preference for a project: UP-01-TC03
+            Given user has created project "test-preference1"
+             When user clicks on user dropdown
+              And user clicks User Preference option
+              And user clicks on Project dropdown on User Preferences page
+              And user searches project "test-preference1"
+              And user selects the project "test-preference1" from the dropdown
+              And user clicks on Add from navigation menu
+             Then user can see "test-preference1" option selected in project namespace
+
+
+        @regression @to-do
+        Scenario: Creating project with project preference: UP-01-TC04
+            Given user has created project "test-preference1"
+              And user is at Add page
+             When user clicks on user dropdown
+              And user clicks User Preference option
+              And user clicks on Project dropdown on User Preferences page
+              And user types project "test-preference2" in search bar
+              And user clicks on the Create project option from the dropdown
+              And user clicks on Add from navigation menu
+             Then user can see "test-preference2" option selected in project namespace
+
+
+        @regression @to-do
+        Scenario: Setting Graph preference for Topology: UP-01-TC05
+             When user clicks on user dropdown
+              And user clicks User Preference option
+              And user is on Topology dropdown on User Preferences page
+              And user selects "Graph" option
+              And user clicks on Topology in navigation menu
+             Then user can see topology graph view
+
+
+        @regression @to-do
+        Scenario: Setting List preference for Topology: UP-01-TC06
+             When user clicks on user dropdown
+              And user clicks User Preference option
+              And user is on Topology dropdown on User Preferences page
+              And user selects "List" option
+              And user clicks on Topology in navigation menu
+             Then user can see topology list view
+
+
+        @regression @to-do
+        Scenario: Setting Form preference for Create/Edit resource method: UP-01-TC07
+             When user clicks on user dropdown
+              And user clicks User Preference option
+              And user is on Create/Edit resource method dropdown on User Preferences page
+              And user selects "Form" option
+              And user clicks on Add in navigation menu
+              And user clicks on Helm charts
+              And user selects "Nodejs v0.0.1" helm chart
+              And user clicks on Install Helm Chart button
+             Then user can see Form view option selected in Install Helm Chart page
```

Comment:
This is true as per the current understanding of the UX. However it may change later when the PR for the related story goes into review. We should go ahead with this now, since this is true as per the UXD and acceptance criteria.

Reply:
In case of change you can update the gherkin as well with your pr